gridstatus/spp.py

```diff
@@ -704,6 +704,70 @@ def _process_operating_reserves(self, df):
 
         return df
 
+    @support_date_range("5_MIN")
+    def get_day_ahead_marginal_clearing_prices(self, date, end=None, verbose=False):
+        """Get Day Ahead Marginal Clearing Prices. Updated every 5 minutes.
```

Comment:
use description from site: "Provides Marginal Clearing Price information by Reserve Zone for each Day-Ahead Market solution for each Operating Day.  Posting is updated each day after the DA Market results are posted."

Reply:
let's name this `get_da_operating_reserve_prices`